packages/kbn-utils/README.md

```diff
@@ -0,0 +1,3 @@
+# @kbn/utils
+
+Shared server-side utilities shared across packages and plugins.
```

Comment:
btw are we sure that we'd never need Kibana-specific browser utils?

Reply:
It's possible but thought it would be best to keep it focused to start.

Reply:
I expect we will quickly start hearing stories of `@kbn/utils` gets pulled into the browser builds accidentally and failing because things like `REPO_ROOT` don't work in the browser (and will throw an error on page load, maybe even at build time).

It's unclear to me what we might do about this though, other than having separate entry points like `@kbn/utils/node` and `@kbn/utils/browser` which have slightly different APIs but share the exports they can. Still, something we can do in a follow-up as needed.